src/app/body/view-organization-details/teams-card/teams-card.component.html

```diff
@@ -15,11 +15,11 @@
     <div class="card-body pt-4">
         <ng-container>
             <div class="row">
-                <div class="col-auto float-left">
+                <div class="col-md-10 float-left">
```

Comment:
Why are we changing this to md-10

Reply:
because title was taking the whole column width when description is long, that is why gear icon was shifting to bottom